apps/web/lib/hooks/useIsBookingPage.ts

```diff
@@ -4,7 +4,9 @@ import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
 
 export default function useIsBookingPage(): boolean {
   const pathname = usePathname();
-  const isBookingPage = ["/booking/", "/cancel", "/reschedule"].some((route) => pathname?.startsWith(route));
+  const isBookingPage = ["/booking", "/cancel", "/reschedule", "/d"].some((route) =>
```

Comment:
what if pathname is `/bookinggggg`? 🤔

Reply:
then we 404!
<img width="1512" alt="Screenshot 2025-02-12 at 5 19 57 AM" src="https://github.com/user-attachments/assets/29f93e54-48bc-4aa7-951f-8e1f4059e7c8" />